### PR TITLE
port to new http server

### LINKED ISSUE #14: Move to corral
Most official ponylang libraries have been moved to https://github.com/ponylang/corral
Lets do so too!

### PR DESCRIPTION
**Pull request by Theodus**
Closes #14

### CODE REVIEW

**Comment by Theodus** [comment]
@mfelsche I've invited you to be a collaborator on this project. Could you review this commit? No rush.

**Comment by mfelsche** [comment]
@Theodus i am flattered! Will do! :)

File: jennet/responder.pony [hunk 1]
```diff
@@ -1,60 +1,54 @@
...
+        ":".join([host; service].values())
+      else
+        "Jennet" // TODO get IP from server
+      end
+
+  fun apply(res: Response, body: ByteArrays, ctx: Context box)
+  =>
+    ctx.session.send(res, body, ctx.request_id)
+    match res.header("Connection")
+    | let h: String if h.contains("close") => ctx.session.dispose()
```
[the remark this answers is not in the document]
Comment (Theodus) [reply jennet/responder.pony:31]: Removing either of these results in 404 in my browser. I'm not sure why.

[the remark this answers is not in the document]
Comment (Theodus) [reply jennet/responder.pony:31]: I've opened an issue for this: #18

**Pull request merged**